modules/nw-configuring-elb-timeouts-aws-classic.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: CONCEPT
+[id="nw-configuring-elb-timeouts-aws-classic_{context}"]
+= Configuring Elastic Load Balancer timeouts for AWS Classic nodes
```

Comment:
I don't know what "AWS Classic nodes" means exactly.  This wording is potentially confusing because AWS has two network models: "EC2-Classic" and "VPC" (virtual private cloud), and this has caused much confusion in the past between "EC2-Classic" and "Classic Load Balancers", especially when [AWS announced the retirement of EC2-Classic Networking](https://aws.amazon.com/blogs/aws/ec2-classic-is-retiring-heres-how-to-prepare/).  To be completely clear:

* OpenShift 4 requires using a VPC.
* OpenShift 4 does not support using EC2-Classic networking (and therefore its retirement has no impact on OpenShift).  
* OpenShift 4 supports both Classic Load Balancers (CLBs) and Network Load Balancers (NLBs), in VPCs.

So I don't think that it is necessary to mention VPCs here, but I'm wary of any use of "Classic" outside of the phrase "Classic Load Balancer" because such uses of "Classic" could be misconstrued as referring to EC2-Classic.  

```suggestion
= Configuring Classic Load Balancer timeouts on AWS
```

Reply:
Thanks for clarifying all this, totally agree: less confusion is better! Heading is updated